mix.exs

```diff
@@ -23,7 +23,7 @@ defmodule WebSockex.Mixfile do
 
   def application do
     applications = [:logger, :ssl, :crypto] ++ applications(otp_release())
-    [applications: applications, mod: {WebSockex.Application, []}]
+    [extra_applications: applications, mod: {WebSockex.Application, []}]
```

Comment:
`applications` -> `extra_applications`.

Since most dependencies are only for testing, the only difference, for now, is if the `telemetry` app will start with WebSockex. I think we should start `telemetry` when starting WebSockex.